### ISSUE TITLE
Could not connect to action server

### ISSUE THREAD

**Issue by jasonvank** [ros-industrial-consortium/descartes#191]
I am using UR modern drivers to connect to my UR10 robot. 

When I run the descartes, it has this error as the picture below.

`[   INFO] [1485417794.536098290]: With 90 - new best score: 8.723923`
`[   INFO] [1485417794.603379840]: score: 8.723923`
`[   INFO] [1485417794.603440975]: FOUND A PATH`
`[   INFO] [1485417794.603440975]: Traj size: 90 list size:90`
`[ERROR] [1485417794.615877032]: Could not connect to action server`
`[ERROR] [1485417794.710848660]: Could not execute trajectory!`
`[ WARN] [1485417794.720803861]: SERVER WANING!!! Attempting to unload library while objects created by this loads attempting to unload the library or destroying the ClassLoader. The library will NOT be unloaded.`
`[tutorial1-1] process has died [pid 1610, exit code 252, cmd /hime/nb06/Documents/Wenkai/new_drivers/devel/lib/descartes/nb06/.ros/log/bd7efadc-e39d-11e6-bleach-f8cab809a25/tutorial1-1.log]`

Could you provide some alternative solution please? Thanks!

**Comment by gavanderhoorn** [comment]
@jasonvank: could you either:

 1. just copy-paste the error text from your console into your issue, or
 1. if you really have to use an image, attach it directly to the issue (GitHub supports this)

But there is really no need to use a screenshot for something that is essentially text.

**Comment by jasonvank** [comment]
@gavanderhoorn Thanks.

I have managed to solve the issue. 

When I run my descartes tutorial on my simulation, the robot moves as expected. However, I am trying to plan using the same solved points but the UR10 robot doesn't move (Yes, there's connection between the robot and ROS as I tried using a plugin planner before to test it). Is there any step I'm missing?  Maybe the joint values are not being published to the ToROSJointTrajectory function?

I'm using the Descartes tutorial.

Jason.
